Cartfile

```diff
@@ -1,4 +1,3 @@
 binary "https://www.mapbox.com/ios-sdk/opencv.json" ~> 2.4.13
 github "mapbox/mapbox-events-ios" ~> 0.9.3
-github "mapbox/mapbox-navigation-ios" ~> 0.33.0
```

Comment:
By removing this explicit dependency, you also remove the ability to declare which versions the code is compatible with. canImport by itself doesn’t have a way to check for library versions. So the library is just as dependent on the navigation SDK as before, but without the safety that comes from the package manager checking for version compatibility.

Reply:
you right, it's a good reason to not use `canImport`